README.md

```diff
@@ -28,103 +28,27 @@ See the [eris-db documentation](https://erisindustries.com/components/erisdb/) f
 
 1. [Install go](https://golang.org/doc/install)
 2. Ensure you have `gmp` installed (`sudo apt-get install libgmp3-dev || brew install gmp`)
-3. `go get github.com/eris-ltd/eris-db/cmd/erisdb`
+3. `go get github.com/eris-ltd/eris-db/cmd/eris-db`
 
 
-To run `erisdb`, just type `$ erisdb /path/to/working/folder`
+To run `eris-db`, just type `$ eris-db serve --work-dir <path to chain directory>`
 
 This will start the node using the provided folder as working dir. If the path is omitted it defaults to `~/.erisdb` 
 
 
 ## Usage
 
-Once the server has started, it will begin syncing up with the network. At that point you may begin using it. The preferred way is through our [javascript api](https://github.com/eris-ltd/eris-db.js), but it is possible to connect directly via HTTP or websocket. The JSON-RPC and web-api reference can be found [here](api)
+Once the server has started, it will begin syncing up with the network. At that point you may begin using it. The preferred way is through our [javascript api](https://github.com/eris-ltd/eris-db.js), but it is possible to connect directly via HTTP or websocket. The JSON-RPC and web-api reference can be found [here](docs/api.md)
 
-### Configuration Files
+## Configuration
 
-Three files are currently required: 
-```
-config.toml
-genesis.json
-priv_validator.json
-```
-while `server_conf.toml` is optional
+See commented template config at [server_config.toml](server_config.toml)
```

Comment:
I want it to be something you can click straight through to with comments. To what would we be referring to here? Where/how does cm create config.toml?